prerendering.bs

```diff
@@ -641,6 +645,12 @@ This stores which client hints each origin has opted into receiving, until it ca
   1. Otherwise, [=map/set=] [=Accept-CH cache=][|origin|] to |hintSet|.
 </div>
 
+<h3 id="interaction-with-clear-site-data-clear-cache">Interaction with Clear-Site-Data Clear Cache</h3>
+
+Modify [=Clear cache for origin=] to add the following step at the end.
+1. [=list/For each=] [=navigable/top-level traversable=] |traversable| in the user agent's [=top-level traversable set=]: 
+  1. If |traversable| is a [=prerendering traversable=], then [=destroy a top-level traversable|destroy=] |traversable|.
```

Comment:
This needs to be scoped to _origin_. You can do that by checking if `|traversable|'s [=navigable/active document=]'s [=Document/origin=] is [=same origin=] with |origin|`.

Reply:
Agreed! Updated to include scoping to origin.